File: common/src/main/java/com/cyanspring/common/transport/IClientSocketService.java
package com.cyanspring.common.transport;

import java.util.List;

import com.cyanspring.common.IPlugin;

public interface IClientSocketService extends IPlugin {
	boolean sendMessage(Object obj);
	boolean addListener(IClientSocketListener listener);
	boolean removeListener(IClientSocketListener listener);
}

File: csharp/CSharpLTS/Common/Transport/IClientSocketService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Common.Basic;

namespace Common.Transport
{
    interface IClientSocketService : IPlugin 
    {
        bool SendMessage(Object obj);
        bool AddListener(IClientSocketListener listener);
        bool RemoveListener(IClientSocketListener listener);
    }
}
